ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/migration/scripts/V3__Rebuild_Acs.scala

```diff
@@ -0,0 +1,760 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Note: package name must correspond exactly to the flyway 'locations' setting, which defaults to 'db.migration'
+package db.migration
+
+import java.io.InputStream
+import java.sql.Connection
+import java.util.Date
+
+import akka.stream.scaladsl.Source
+import akka.NotUsed
+import anorm.SqlParser.{str, _}
+import anorm.{BatchSql, Macro, NamedParameter, RowParser, SQL, SqlParser}
+import com.digitalasset.daml.lf.data.Ref
+import com.digitalasset.daml.lf.data.Relation.Relation
+import com.digitalasset.daml.lf.engine.Blinding
+import com.digitalasset.daml.lf.transaction.Transaction
+import com.digitalasset.daml.lf.transaction.Node.{GlobalKey, KeyWithMaintainers}
+import com.digitalasset.daml.lf.value.Value.{AbsoluteContractId, ContractId}
+import com.digitalasset.ledger.backend.api.v1.RejectionReason
+import com.digitalasset.ledger.backend.api.v1.RejectionReason._
+import com.digitalasset.platform.sandbox.services.transaction.SandboxEventIdFormatter
+import com.digitalasset.platform.sandbox.stores._
+import com.digitalasset.platform.sandbox.stores.ledger.LedgerEntry
+import com.digitalasset.platform.sandbox.stores.ledger.sql.dao.Contract
+import com.digitalasset.platform.sandbox.stores.ledger.sql.serialisation.{
+  ContractSerializer,
+  KeyHasher,
+  TransactionSerializer,
+  ValueSerializer
+}
+import com.google.common.io.ByteStreams
+import org.flywaydb.core.api.migration.{BaseJavaMigration, Context}
+import org.slf4j.LoggerFactory
+
+import scala.collection.immutable
+
+/**
+  * V1 was missing divulgence info
+  * V2 adds corresponding new tables
+  * V3 fills the new tables
+  */
+class V3__Rebuild_Acs extends BaseJavaMigration {
```

Comment:
I suggest you "group" the migration with minor number. that is using

- `V2_0__Contract_divulgence.sha256` 

- `V2_1__Rebuild_Acs`

Reply:
Done.

Reply:
:+1: